manifests/init.pp

```diff
@@ -94,6 +98,8 @@
 class nrpe (
   Array[Stdlib::Host]                  $allowed_hosts                   = ['127.0.0.1'],
   Stdlib::IP::Address                  $server_address                  = '0.0.0.0',
+  Hash                                 $commands                        = {},
```

Comment:
This should work as well?
```suggestion
  Hash[String[1],Hash[String[1],String[1]]] $commands                   = {},
```

Reply:
maybe?  I don't think I want to start using something quite so ugly everywhere though.  Maybe submit a PR for a new type? `Stdlib::Resources`??

Reply:
@bastelfreak So are you ok merging this as is?